aws/websites/metrics.pytorch.org/install.yml

```diff
@@ -23,7 +26,15 @@
           /etc/pytorch/dashboards/ \
           /etc/pytorch/grafana
 
-          find /etc/pytorch -type d | xargs chmod 777
+        wget https://github.com/marcusolsson/grafana-json-datasource/releases/download/v{{json_data_source_ver}}/marcusolsson-json-datasource-{{json_data_source_ver}}.zip
+        unzip marcusolsson-json-datasource-{{json_data_source_ver}}.zip -d /etc/pytorch/grafana/plugins
+        rm marcusolsson-json-datasource-{{json_data_source_ver}}.zip
+
+        wget https://rockset-public.s3-us-west-2.amazonaws.com/rockset-backend-datasource-{{rockset_data_source_ver}}.zip
```

Comment:
This is more a highlevel question--why do we still want the Rockset integration if the other one is much more preferred?